### PR TITLE
update readme for CrossEntropyLoss_cn.rst and cross_entropy_cn.rst

### PR DESCRIPTION
**Pull request by chajchaj**
升级paddle2.0中nn/layers/loss.py中新增CrossEntropyLoss 和 nn/functional/loss.py中新增cross_entropy的对应中文文档。

### CODE REVIEW

**Comment by chajchaj** [comment]
Lce1
Lce2
Lce3
Lce4
Lce5

File: doc/paddle/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.rst [hunk 1]
```diff
@@ -5,35 +5,113 @@ CrossEntropyLoss
...
+该OP实现了softmax交叉熵损失函数。该函数会将softmax操作、交叉熵损失函数的计算过程进行合并，从而提供了数值上更稳定的计算。
 
-如果提供 `weight` 参数的话，它是一个 `1-D` 的tensor, 每个值对应每个类别的权重。
-该损失函数的数学计算公式如下：
+该OP默认会对结果进行求mean计算, 您也可以影响该默认行为， 具体参考reduction参数说明。
 
-    .. math::
-        loss_j =  -\text{input[class]} +
-        \log\left(\sum_{i=0}^{K}\exp(\text{input}_i)\right), j = 1,..., K
+该OP可用于计算硬标签或软标签的交叉熵。其中，硬标签是指实际label值，例如：0, 1, 2...，软标签是指实际label的概率，例如：0.6, 0,8, 0,2...
```
Comment (jzhang533) [inline doc/paddle/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.rst:12]: 这一句我觉得可以写的更清楚一些。
尽管我也还没想到更合适的描述方式，但是，但看这一句，担心对于读者理解起来还是有些困难。

File: doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst [hunk 2]
```diff
@@ -48,3 +118,5 @@ cross_entropy
         loss = paddle.nn.functional.cross_entropy(input=input_data, label=label_data, weight=weight_data)
         print(loss)
         # [4.38418674]
+
```
Comment (jzhang533) [inline doc/paddle/api/paddle/nn/functional/loss/cross_entropy_cn.rst:121]: 示例代码是否可以补充一个softlabel时的示例。
